auth.rb

```diff
@@ -1,14 +1,10 @@
 class Authentication
-  # please improve this rails action,
-  # if you'd like to put some code in another file please state so:
-  # authentication API
   def auth
-    user = User.find(params[:username])

```

Comment:
`find` will try to find by id, assuming `params[:username]` should be string of username so it will raise `ActiveRecord::RecordNotFound` error. Better to use `find_by` that will return `nil` instead.